etl/steps/data/garden/papers/2023-12-12/smil_2017.yml

```diff
@@ -1,43 +1,37 @@
+definitions:
+  common:
```

Comment:
The filename here is missing 'meta', I think that will prevent it being used?

Reply:
Good catch, indeed, the metadata was ignored!